### PR TITLE
Update homematicip_cloud.markdown

### PR DESCRIPTION
**Pull request by saepfle**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by saepfle** [comment]
Update of current documentation - please set status for "current"

**Comment by steckenpferd** [comment]
Dear mods aproved from my site please continue

**Review by steckenpferd — DISMISSED** [review]
Approved

File: source/_components/homematicip_cloud.markdown [hunk 1]
```diff
@@ -41,6 +41,7 @@ There is currently support for the following device types within Home Assistant:
 * Light
 * Sensor
 * Switch
+* Shutter
```
Comment (klaasnicolaas) [inline source/_components/homematicip_cloud.markdown]:
```suggestion
* Cover
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by steckenpferd** [comment]
When will it be live?

**Comment by klaasnicolaas** [comment]
Only after the conflict has been resolved @steckenpferd

**Comment by saepfle** [comment]
Not sure who can resolve the "conflict" - and don't see any - descriptions were corrected by me accoring this homepage: https://github.com/coreGreenberet/homematicip-rest-api

**Comment by saepfle** [comment]
Sorry did not see the to does in the conflict - hopefully its ok now and documentation can finally be updated

**Comment by klaasnicolaas** [comment]
After the deploy is ready, we can merge this 😉

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Thnx for the PR! 👍

**Pull request merged**